src/Nokia_LCD.cpp

```diff
@@ -14,9 +14,18 @@
 namespace {
 const uint8_t kDisplay_max_width = 84;
 const uint8_t kDisplay_max_height = 48;
+
+// Instantiate the default font
+const LcdFont nokiaFont { 
+    [](char c) { return Nokia_LCD_Fonts::kDefault_font[c - 0x20]; }, 
+    Nokia_LCD_Fonts::kColumns_per_character,
+    Nokia_LCD_Fonts::hSpace, 
+    1 
+};
+
 // Each row is made of 8-bit columns
 const unsigned int kTotal_rows =
-    kDisplay_max_height / Nokia_LCD_Fonts::kRows_per_character;
+    kDisplay_max_height / nokiaFont.rowSize;
```

Comment:
Hmmm, is this correct with a different font, i.e. a smaller one? I guess if we use a smaller font then we could have more rows?
Should this perhaps be part of the `LcdFont` class instead? Then the way we calculate `kTotal_bits` may be reconsidered.

I may not have thought this through completely. :thinking:

Reply:
It's correct, because even the small font uses 8 bit height (remember rowSize = 8). Of course, it's possible to use less (or more) bits than 8 for a single row like the Adafruit implementation, but it involves a more complex refactoring on the `printCharacter()` and other methods. So, in short, all fonts have same height and different width. 

> Should this perhaps be part of the LcdFont class instead?
I though the same while coding, but I need a static constant there and use it like `LcdFont::ROWS_PER_CHARACTER`. If you feel ok with this, I can change it.

![IMG_20210724_151815](https://user-images.githubusercontent.com/7151022/126881921-01037cea-e84d-46ba-97d4-f717a524e37d.jpg)

Reply:
Yes, let's leave it as it was and use the constant here so not to give the false impression that this depends on the default font.